spec/lib/quiet_open_spec.rb

```diff
@@ -0,0 +1,26 @@
+# -*- encoding : utf-8 -*-
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
+
+describe "quietly_try_to_open" do
+
+  let(:fakeController) { double(ApplicationController) }
```

Comment:
Hmm, looks almost like I carelessly renamed it (and `emptyStream`) as an afterthought, or I had an oddly formatted example open in another window. No idea why I did that. Distractingly odd. Will fix.